schema/geometry_schema.json

```diff
@@ -94,6 +94,38 @@
 						},
 						"north_axis": {
 							"type": "number"
+						},
+						"ground": {
+							"type": "object",
+							"properties": {
+								"floor_offset": {
+									"description": "Specifies the vertical offset between the building floor and the ground plane. Use a value < 0 for buildings with below-grade spaces, or a value > 0 for, e.g., buildings on piers.",
```

Comment:
Can we expand "Specifies the vertical offset between the bottom floor surface of the lowest story and the ground plane at x = 0 and y = 0. Use a value < 0 for buildings with below-grade spaces, or a value > 0 for, e.g., buildings on piers."

Reply:
The user can set the origin x, y where they want, I think this can remove the need for the x_shift and y_shift properties

Reply:
Unless you are thinking there are multiple ground objects in a project?